### DISCUSSION TITLE
Page not found 404 error

### DISCUSSION THREAD

**Discussion by chandiran-v** [community/community#75981] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Here is my repo: https://github.com/chandiran-v/weather-app/
My live page from this repo: https://chandiran-v.github.io/weather-app/

Please take a look and fix this. I make this using react js. 
My index.html file is inside public folder.

**Comment by ghost** [comment]
Hello @chandiran-v 

your weather-app works fine for me:
![image](https://github.com/community/community/assets/101295551/332168bd-3ae2-491f-8065-92ad4b55dfa5)


can you describe your problem more clearlly? or check if it is still reproducable for you and if so provide steps for us to reproduce it.

**Reply by chandiran-v** [reply]
Yes. I am using React js.  I have deleted rel=manifest file from html. Then I figured out and added. Thanks for you concern.

**Comment by chandiran-v** [comment, marked as answer]
Added link tag: <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />

**Reply by chandiran-v** [reply]
<link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
